src/gpodder/youtube.py

```diff
@@ -44,33 +44,33 @@
     # Fallback to an MP4 version of same quality.
     # Try 34 (FLV 360p H.264 AAC) if 18 (MP4 360p) fails.
     # Fallback to 6 or 5 (FLV Sorenson H.263 MP3) if all fails.
-    (46, ([46, 37, 45, 22, 44, 35, 43, 18, 6, 34, 5],
+    (46, ([46, 37, 45, 22, '136+140', 44, 35, 43, 18, '134+140', 6, 34, 5],
```

Comment:
The same format shouldn't appear multiple time, should it?

Reply:
They aren't the same formats, one is 13-SIX and the other is 13-FOUR. 136 is the video for 22 and 134 is video for 18, both use the same audio.

Reply:
you're right: my mistake